scripts/proteinpaint_txt_coding_ssm.R

```diff
@@ -0,0 +1,78 @@
+# create ProteinPaint txt files of GAMBLR bundled coding SSMs, from grch37 and 
+# hg38 projections, genome and capture seq types, from lymphoma genes. 
+
+
+library(GAMBLR.utils)
+library(GAMBLR.data)
+library(dplyr)
+
+
+### grch37
+
+# genome
+maf_37_g = get_ssm_by_samples(
+  projection = "grch37",
+  this_seq_type = "genome", 
+  basic_columns = FALSE
+)
+pp_37_g <- ssm_to_proteinpaint(
+  maf_data = maf_37_g,
+  this_seq_type = "genome",
+  sample_type = "time_point",
```

Comment:
Can you use pairing_status instead of time_point?

Reply:
Sure! However, since this is the bundled data, the pairing_status of all samples are `matched`.
```
> library(GAMBLR.data)
> meta <- get_gambl_metadata()
Using the bundled metadata in GAMBLR.data...
> table(meta$pairing_status)
matched 
    300 
```

Reply:
Is this because you're only generating these files for genome data (not capture)?

Reply:
Doesn't this return quite a few rows?
```
meta <- get_gambl_metadata(seq_type_filter="capture")
```

Reply:
Isn't the Reddy cohort in the bundled results? All of those are unmatched.